### ISSUE TITLE
not playing on my site

### ISSUE THREAD

**Issue by bitsbb01** [telase/Web-IPTV-Player#4]
it works fine at http://iptvde.mypressonline.com/
however at my own site, with all the same files, nothing plays. 
Theres not even a play button, is there something else i need to do to the server?

**Comment by telase** [comment]
What is your website address?
maybe there is a malfunction in your browser

**Comment by bitsbb01** [comment]
http://ofciptv.alienwarez.org/

**Comment by telase** [comment]
NO:
<li class="nav-item">
<a class="channel nav-link" data-value="https://raw.githubusercontent.com/bitsbb01/IPTV-M3US/master/m3us/FlixHD.m3u" href="javascript:void(0);">Test Video</a>
</li>

YES:
<li class="nav-item">
<a class="channel nav-link" data-value="https://y5w8j4a9.ssl.hwcdn.net/andflixhd/tracks-v1a1/index.m3u8" href="javascript:void(0);">Test Video</a>
</li>

IPTV links will be m3u8, not m3u links
You will enter the links one by one

**Comment by bitsbb01** [comment]
ok, changed it so now theres 2 test files, 1 m3u8 and 1 mp4, it still not loading it, think its gotta be a php addon in godaddy.. lol

**Comment by telase** [comment]
noooooooo
not link
https://raw.githubusercontent.com/bitsbb01/IPTV-M3US/master/m3us/FlixHD.m3u8

there will be a direct link
this:
https://y5w8j4a9.ssl.hwcdn.net/andflixhd/tracks-v1a1/index.m3u8

**Comment by telase** [comment]
Look
http://iptvde.mypressonline.com/usa.php
I added your link to the first channel

**Comment by bitsbb01** [comment]
ahhh, ok... lol sorry.. i see what u mean.. not github.. 

but still not showing

**Comment by telase** [comment]
So this is what I can see from here
I can't say anything without seeing ftp
sorry

**Comment by bitsbb01** [comment]
yep, i understand. its gotta be a php addon.. weird thing is, it works fine on a free host, but on godaddy it doesnt.. lol

**Comment by bitsbb01** [comment]
i've created u an ftp to the sub domain where the site is, if you goto http://www.alienwarez.org/ and set up an account, i'll send u a message there wiht the details.. that way can make sure its you.. lol

**Comment by telase** [comment]
i wrote there
"I am telase in github"

**Comment by bitsbb01** [comment]
sent u the details there...
